### PR TITLE
Adds datatypes to swift definition

### PR DESCRIPTION
**Pull request by SerekKiri**
This pull request adds data types to swift language definition.

### CODE REVIEW

File: data/DataType/float_swift.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+keyword: float
+integer-min: '1.2E-38'
```
Comment (siddhpant) [inline data/DataType/float_swift.yaml]: This is the minimum *positive* value, not the minimum value.

File: data/DataType/double_swift.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+keyword: double
+integer-min: '2.3E-308'
```
Comment (siddhpant) [inline data/DataType/double_swift.yaml]: This is the minimum *positive* value, not the minimum value.

**Comment by siddhpant** [comment]
:+1: :+1: :+1:

File: data/DataType/double_swift.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+keyword: double
+integer-min: '1.7976931348623157e+308'
+integer-max: '-1.7976931348623157e+308'
+integer-signed: true
```
Comment (siddhpant) [inline data/DataType/double_swift.yaml]: Lol

Comment (SerekKiri) [reply data/DataType/double_swift.yaml]: Fixed 😄

File: data/Language/Swift.yaml [hunk 4]
```diff
@@ -15,6 +15,15 @@ delimiters:
   - round_braces
   - square_braces
   - curly_braces
+datatypes:
+  - bool_swift
```
Comment (jayvdb) [inline data/Language/Swift.yaml]: this is the same as many other `bool` datatype definitions.  We only need one.  it should be called bool_lower_case , similar to bool_title_case in https://github.com/coala/coAST/pull/21

And all other lang defs should refer to it if they have the same data type

File: data/Language/Jinja2.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+identifier: Jinja2
```
Comment (jayvdb) [inline data/Language/Jinja2.yaml]: why is this in a patch about Swift?

Comment (SerekKiri) [reply data/Language/Jinja2.yaml]: I merged branch

**Comment by jayvdb** [comment]
Not rebased properly

File: data/DataType/int64.yaml [hunk 6]
```diff
@@ -1,4 +1,4 @@
-keyword: int64
+keyword: Int64
```
Comment (siddhpant) [inline data/DataType/int64.yaml]: Does golang allow capitalising the first letter of a keyword?

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/VisualBasic.yaml, line 98.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/Pascal.yaml, line 53.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/VisualBasic.yaml, line 98.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/Pascal.yaml, line 53.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/VisualBasic.yaml, line 98.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/Pascal.yaml, line 53.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/VisualBasic.yaml, line 98.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/Pascal.yaml, line 53.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/VisualBasic.yaml, line 98.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/Pascal.yaml, line 53.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/VisualBasic.yaml, line 98.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
Comment on 1bb2849b9f12f160d0db3f52e4efe8dbda32c1d6, file data/Language/Pascal.yaml, line 53.

truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

File: data/Language/Java.yaml [hunk 7]
```diff
@@ -14,7 +14,7 @@ delimiters:
   # string delimiters:
   - double_quote_slash_escape
 datatypes:
-  - boolean_java
+  - bool_lower_case
```
Comment (jayvdb) [inline data/Language/Java.yaml]: the primative in Java is called `boolean`

File: data/Language/Golang.yaml [hunk 8]
```diff
@@ -49,14 +49,14 @@ datatypes:
...
-  - uint64_go
-  - int8_go
-  - int16_go
-  - int32_go
-  - int64_go
+  - uint8
+  - uint16
+  - uint32
+  - uint64
+  - int8
```
Comment (jayvdb) [inline data/Language/Golang.yaml]: golang also has `int`, but that can be a separate bug

File: data/DataType/boolean_java.yaml [hunk 9]
```diff
@@ -1,7 +0,0 @@
-keyword: boolean
```
Comment (jayvdb) [inline data/DataType/boolean_java.yaml]: this was correct.

File: data/DataType/boolean_java.yaml [hunk 10]
```diff
@@ -4,4 +4,4 @@ integer-max: '1'
 integer-signed: false
 literals:
   - 'true'
-  - 'false'
+  - 'false'
```
Comment (jayvdb) [inline data/DataType/boolean_java.yaml]: no new line character at the end of file (new-line-at-end-of-file)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/58/commits/24801c2965a0bda11d1312764f6df326ce00affb

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Pull request merged**